src/Microsoft.ML.Data/Evaluators/BinaryClassifierEvaluator.cs

```diff
@@ -787,6 +781,246 @@ private void ComputePrCurves()
                 }
             }
         }
+
+        /// <summary>
+        /// Evaluation results for binary classifiers, excluding probabilistic metrics.
+        /// </summary>
+        public class Result
+        {
+            /// <summary>
+            /// Gets the area under the ROC curve.
```

Comment:
>Gets [](start = 16, length = 4)

not necessary on this PR, but maybe we can provide links to the longer explanation of the metrics (wikipedia or the MSFT doc on RML, maybe?), since those comments will be the user-facing documentation about the metrics (I assume)

Reply:
I agree this might be helpful. Maybe we can review later, frankly I don't quite know how to insert links in XML docs since AFAIK `<a` is not a supported tag.

---
In reply to: [216470620](https://github.com/dotnet/machinelearning/pull/869#discussion_r216470620) [](ancestors = 216470620)